modules/post-processing/yolo-object-detection/modulestack.yaml

```diff
@@ -0,0 +1,55 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: This template deploys a Module specific IAM permissions
+
+Parameters:
+  DeploymentName:
+    Type: String
+    Description: The name of the deployment
+  ModuleName:
+    Type: String
+    Description: The name of the Module
+  RoleName:
+    Type: String
+    Description: The name of the IAM Role
+
+Resources:
+  Policy:
+    Type: "AWS::IAM::Policy"
+    Properties:
+      PolicyDocument:
+        Statement:
+          - Action:
+              - "s3:Create*"
+              - "s3:Put*"
+              - "s3:Delete*"
+              - "s3:Get*"
+              - "s3:List*"
+            Effect: Allow
+            Resource: ["arn:aws:s3:::addf*", "arn:aws:s3:::cdk*"]
+          - Action:
+              - "secretsmanager:Get*"
+              - "secretsmanager:DescribeSecret"
+              - "secretsmanager:ListSecretVersionIds"
+              - "kms:Decrypt*"
+            Effect: Allow
+            Resource: !Sub "arn:aws:secretsmanager:${AWS::Region}:${AWS::AccountId}:secret:*"
```

Comment:
this is probably too permissive. it grants our CodeBuild execution permission to read all secrets in the account/region. this should probably be scoped to either an addf- prefix, or use a parameter passed from the manifest to the modulestack.yaml

Reply:
K will update to be addf*

Reply:
If this is being added in order to make use of the dockerCredentialsSecret then it can probably be removed from here completely. seed-farmer automatically adds an inline policy to the Module Role with permissions on the dockerCredentialsSecret if it is specified in the deployment.yaml

Reply:
updated